src/engine/source/builder/builders/opBuilderHelperFilter.cpp

```diff
@@ -1420,11 +1377,10 @@ base::Expression opBuilderHelperIsFalse(const std::any& definition)
     // Tracing
     const auto successTrace {fmt::format("[{}] -> Success", name)};
 
-    const auto failureTrace {fmt::format("[{}] -> Failure:", name)
-                             + "Field \"{}\" is true"};
+    const std::string failureTrace {fmt::format("[{}] -> Failure: Target field '{}' is true", name, targetField)};
 
-    const auto failureMissingValueTrace {
-        fmt::format("[{}] -> Failure: Field \"{}\" not found", name, targetField)};
+    const std::string failureMissingValueTrace {
+        fmt::format("[{}] -> Failure: Target field '{}' not found", name, targetField)};
```

Comment:
all this trace changes where due to the issue on replacing a field in runtime? because there's no additional method inside HelperFilter

Reply:
Yes, the fixes on the trace go beyond this PR scope